### ISSUE TITLE
False positives when using imports from Cython libraries

### ISSUE THREAD

**Issue by jurf** [PyCQA/pylint#3740]
### Steps to reproduce
1. Install [`kivy`](https://pypi.org/project/Kivy/) using Python 3.7
2. Run pylint on this file:
```
import kivy
from kivy.properties import StringProperty
```

Original issue: https://github.com/kivy/kivy/issues/6583

### Current behavior
pylint reports `No name 'StringProperty' in module 'kivy.properties'`.

### Expected behavior
No such error is reported, as `StringProperty` is a correct class name.

### pylint --version output
```
$ pylint --version output
pylint 2.5.3
astroid 2.4.2
Python 3.7.7 (default, Jun  3 2020, 15:16:31) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)]
```
Fedora 32

**Comment by jurf** [comment]
This also happens with `EventDispatcher` members. I did some digging around, according to this [StackOverflow answer](https://stackoverflow.com/a/61567717/6199517), it’s because those parts are Cython libraries.

**Comment by jurf** [comment]
Seems to be a duplicate of https://github.com/PyCQA/pylint/issues/1524.
